client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -588,6 +588,12 @@ function getFallbackDestination( {
 		const emails = titanProducts[ 0 ].extra?.email_users;
 		if ( emails && emails.length > 0 ) {
 			debug( 'site with titan products' );
+			if ( cart?.products?.length === 1 ) {
+				const domain = titanProducts[ 0 ]?.meta;
+				if ( domain ) {
+					return `/email/${ domain }/manage/${ siteSlug }?new-email=${ emails[ 0 ].email }`;
```

Comment:
```suggestion
					return getEmailManagementPath( siteSlug, domain, null, { 'new-email': emails[ 0 ].email } );
```
For consistency, we should use this function from `client/my-sites/email/paths.ts`.